### PR TITLE
Feature/crud for product

### PR DESCRIPTION
**Pull request by eeveesharp**

### CODE REVIEW

File: Shop/Shop/Controllers/ProductController.cs [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+            return Ok();
+        }
+
+        [HttpPost]
+        public IActionResult Create()
+        {
+            using (ApplicationContext db = new ApplicationContext())
+            {
+                Product phone = new Product { Name = "Phone", Price = 123, Quantity = 33 };
```
Comment (stas1408123) [inline Shop/Shop/Controllers/ProductController.cs:63]: Delete hard code

File: Shop/Shop/Controllers/ProductController.cs [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+                db.Products.Update(product);
+
+                db.SaveChanges();
+            }
+
+            return Ok();
+        }
+
+        [HttpPost]
+        public IActionResult Create()
```
Comment (stas1408123) [inline Shop/Shop/Controllers/ProductController.cs:59]: Add input data

File: Shop/Shop/Controllers/ProductController.cs [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+            using (ApplicationContext db = new ApplicationContext())
+            {
+                resultProduct = db.Products.ToList();
+            }
+
+            return resultProduct; 
+        }
+
+        [HttpDelete]
+        public IActionResult Delete()
```
Comment (stas1408123) [inline Shop/Shop/Controllers/ProductController.cs:31]: Add input data

File: Shop/Shop/Controllers/ProductController.cs [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+        {
+            using (ApplicationContext db = new ApplicationContext())
+            {
+                Product product = db.Products.FirstOrDefault();
```
Comment (stas1408123) [inline Shop/Shop/Controllers/ProductController.cs:35]: What data you whant get?

File: Shop/Shop/Controllers/ProductController.cs [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+    {
+        private readonly ILogger<ProductController> _logger;
+
+        public ProductController(ILogger<ProductController> logger)
+        {
+            _logger = logger;
+        }
+
+        [HttpGet]
+        public List<Product> Get()
```
Comment (stas1408123) [inline Shop/Shop/Controllers/ProductController.cs:18]: Rename method to GetAll

**Review by stas1408123 — CHANGES_REQUESTED** [review]

File: Shop/Shop/Controllers/ProductController.cs (hunk 5 above)
Comment (eeveesharp) [reply Shop/Shop/Controllers/ProductController.cs:18]: fixed

File: Shop/Shop/Controllers/ProductController.cs (hunk 3 above)
Comment (eeveesharp) [reply Shop/Shop/Controllers/ProductController.cs]: fixed

File: Shop/Shop/Controllers/ProductController.cs (hunk 2 above)
Comment (eeveesharp) [reply Shop/Shop/Controllers/ProductController.cs]: fixed

File: Shop/Shop/Controllers/ProductController.cs (hunk 1 above)
Comment (eeveesharp) [reply Shop/Shop/Controllers/ProductController.cs]: fixed

**Review by stas1408123 — APPROVED** [review]

**Review by nantonov — APPROVED** [review]

**Review by Blackolf931 — APPROVED** [review]

**Pull request merged**